src/mann_whitney.jl

```diff
@@ -139,10 +139,11 @@ function pvalue(x::ExactMannWhitneyUTest; tail=:both)
         # cannot account for ties
         if tail == :both
             if x.U < x.nx * x.ny / 2
-                2 * pwilcox(x.U, x.nx, x.ny, true)
+                p = pwilcox(x.U, x.nx, x.ny, true)
             else
-                2 * pwilcox(x.U - 1, x.nx, x.ny, false)
+                p = pwilcox(x.U - 1, x.nx, x.ny, false)
             end
```

Comment:
Maybe you could write
```julia
            p = if x.U < x.nx * x.ny / 2
                pwilcox(x.U, x.nx, x.ny, true)
            else
                pwilcox(x.U - 1, x.nx, x.ny, false)
            end
```
instead (can't make a suggestion on github it seems)? I think it is a bit more readable but I don't think it makes a difference for the compiler (maybe it does?), so it's just a matter of taste I guess.